exporter/exporterhelper/README.md

```diff
@@ -28,24 +28,19 @@ The following configuration options can be modified:
 
 ### Persistent Queue
 
-**Status: under development**
+**Status: [alpha]**
 
-> :warning: The capability is under development and currently can be enabled only in OpenTelemetry
-> Collector Contrib with `enable_unstable` build tag set. 
+> :warning: The capability is under development. To use it, an [filestorage] extension needs to be setup, which requires OpenTelemetry Collector Contrib
 
-With this build tag set, additional configuration option can be enabled:
+To use persistent queue, following setting needs to be enabled:
 
 - `sending_queue`
-  - `persistent_storage_enabled` (default = false): When set, enables persistence via a file storage extension
-    (note, `enable_unstable` build tag needs to be enabled first, see below for more details)
+  - `storage` (default = none): When set, enables persistence and uses the component specified as a storage extension for persistent queue
+  - `persistent_storage_enabled` (default = false): When set, enables persistence using the only available storage extension (fails when no storage extension is enabled or more than one storage extension is available)
```

Comment:
Is this setting necessary? If the `storage` is unspecified the persistent queue is disabled. Isn't that enough for control? Or do we want to have this so that we can enable the storage without knowing the name of the extension?

Alternate suggestion: can we delete `persistent_storage_enabled` setting and make `*` (or some other character sequence) a special value for `storage` setting which finds the only available storage extension?

Reply:
`storage` is a new setting which allows to explicitly select which storage extension to use. However, when only only storage extension is specified in the config, we want to be able to automatically attach it and having `storage: ...` specified is not needed. It would be still helpful though to have control if file-backed or memory-backed queue is used. Hence `persistent_storage_enabled`. I think any suggestions on how to make it more elegant are more than welcome but I think we want to retain such kind of functionality